lib/Parser.js

```diff
@@ -297,6 +297,44 @@ Parser.prototype.initializeEvaluating = function() {
 			}
 			return new BasicEvaluatedExpression().setString(result).setRange(expr.range);
 		});
+		this.plugin("evaluate TemplateLiteral", function(node) {
+			var i, quasis = node.quasis, exprs = [];
+			for (i = 0; i < quasis.length; i++) {
+				exprs.push(new BasicEvaluatedExpression().setString(quasis[i].value.cooked).setRange(quasis[i].range));
+
+				if (i > 0) {
```

Comment:
Could you move this part into a common function? Just define the function before line 300.